dev/com.ibm.tx.core/src/com/ibm/tx/jta/impl/FailureScopeController.java

```diff
@@ -12,18 +12,19 @@
  *******************************************************************************/
 import java.security.AccessController;
 import java.security.PrivilegedAction;
+import java.util.Collections;
 import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
 
 import javax.transaction.SystemException;
 
 import com.ibm.tx.TranConstants;
-import com.ibm.tx.util.ConcurrentHashSet;
-import com.ibm.tx.util.logging.FFDCFilter;
-import com.ibm.tx.util.logging.Tr;
-import com.ibm.tx.util.logging.TraceComponent;
+import com.ibm.websphere.ras.Tr;
```

Comment:
These imports shouldn't have changed.  They should use the com.ibm.tx.util.logging ones for transaction code.  Similar in other files as well.

Reply:
Based off of Jon's changes in PR #17334 last month, this code is accurate.